docs/mobile.md

```diff
@@ -0,0 +1,231 @@
+# Mobile Testing
+
+CodeceptJS allows to test mobile and hybrid apps in a similar manner web applications are tested.
+Such tests are executed using [Appium](http://appium.io) on emulated or physical devices.
+
+What makes CodeceptJS better for mobile testing?
+Take a look. Here is the sample test for a native mobile application written in CodeceptJS:
+
+```js
+I.seeAppIsInstalled("io.super.app");
+I.click('~startUserRegistrationCD');
+I.fillField('~email of the customer', 'Nothing special'));
+I.see('davert@codecept.io', '~email of the customer'));
+I.clearField('~email of the customer'));
+I.dontSee('Nothing special', '~email of the customer'));
+I.seeElement({
+  android: 'android.widget.Button',
+  ios: '//UIAApplication[1]/UIAWindow[1]/UIAButton[1]'
+});
+```
+This test is easy to read and write. Also it will work both on iOS and Android devices.
+Doesn't it sound cool?
+
+## Setting Up
+
+Ensure that you have [CodeceptJS installed](http://codecept.io/installation/).
+You will also need to install [Appium](http://appium.io/). This can be done via npm:
+
+```
+npm i -g appium
```

Comment:
Appium installation is not so easy 😉 I suggest to add here very helpfull tool appium-doctor (npm i -g appium-doctor)
After running it you can check, that everything is OK with Appium